deployment/pcm/templates/podmonitor.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.podMonitor }}
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "pcm.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "pcm.labels" . | nindent 4 }}
+    app.kubernetes.io/component: metrics
+    jobLabel: pcm
+spec:
+  attachMetadata:
+    node: true
+  jobLabel: jobLabel
+  namespaceSelector:
+    matchNames:
+    - {{ .Release.Namespace }}
+  podMetricsEndpoints:
+  # requires hostPort to be set {{ required "A valid .Values.hostPort is required with PodMonitor enabled " .Values.hostPort }}
+  - enableHttp2: false
+    filterRunning: true
+    followRedirects: false
+    honorLabels: true
+    honorTimestamps: true
+    path: /metrics
+    port: pcm-metrics
+    interval: 1s
```

Comment:
Might be worth making `interval` something folks could adjust via `values.yaml`.

Reply:
done - new value `podMonitorInterval`, defaulting to 30s added to values file